scss/_patterns_code-numbered.scss

```diff
@@ -1,44 +1,20 @@
 @import 'settings';
-$sidebar-width: 4.5rem !default;
+$sidebar-width: 3rem !default;
 
 @mixin vf-p-code-numbered {
-  .p-code-numbered {
-    counter-reset: line-numbering;
-    padding: $sph-inner--small 0;
-  }
-
-  .p-code-numbered__line {
-    display: inline-block;
-    padding: 0 $sph-inner 0 ($sidebar-width + $sph-inner);
-    position: relative;
-    white-space: pre-wrap;
-    width: 100%;
-
-    &:empty {
-      display: block;
-      min-height: $sp-xx-large;
+  @include deprecate('3.0.0', 'p-code-numbered is being removed, use the code snippet block numbered variant instead.') {
+    .p-code-numbered {
+      counter-reset: line-numbering;
+      padding: $sph-inner--small 0;
```

Comment:
Shouldn't code numbered have same padding as default code block? we don't need to 0 the padding for the white background anymore.